spce/ibi_espresso/spce.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import print_function
+import time
+import numpy as np
+
+import espressomd
+from espressomd.io.writer import h5md
+import MDAnalysis as mda
+
+
+def write_data(file_name, time, energy, n_part):
+    with open(file_name, 'a') as f:
+        e = energy['total'] / n_part
+        e_pot = (energy['total'] - energy['kinetic']) / n_part
+        e_kin = e - e_pot
+        print(" {} {} {} {}".format(time, e, e_pot, e_kin), file=f)
+
+
+def calc_temperature(system):
+    E = system.analysis.energy()['total']
+    n_part = len(system.part)
+
+    return 2. / 3. * E / n_part
+
+# check for necessary feature
+required_features = ["MASS", "TABULATED"]
+espressomd.assert_features(required_features)
+
+print("Program Information:\n{}\n".format(espressomd.features()))
+
+# set system properties:
+skin = 0.5
+time_step = 0.002
+friction = 5.0
+int_steps = 900
+eq_steps = 100
+
+in_atoms = mda.Universe('spce.gro')
```

Comment:
@fweik do you support a file format. e.g. pdb, we could read without MDanalysis? MDanalysis just seems a bit heavy.